lib/Interpreter/CIFactory.cpp

```diff
@@ -492,7 +492,7 @@ namespace {
       }
 
       static const char *CppInclQuery =
-        "echo | LC_ALL=C " LLVM_CXX " -xc++ -E -v - 2>&1 >/dev/null "
+        "echo | LC_ALL=C " LLVM_CXX " -xc++ -E -v /dev/null 2>&1 "
```

Comment:
Why do you think this is broken? The `-` after `-v` means read output from the pipe, i.e. compile an empty cpp file.